### PR TITLE
aws.ecr.Repository: restore the default to auto-delete images by default

### PR DESCRIPTION
**Pull request by stack72**

### CODE REVIEW

File: provider/cmd/pulumi-resource-aws/schema.json [hunk 1]
```diff
@@ -279467,7 +279467,8 @@
                 },
                 "forceDelete": {
                     "type": "boolean",
-                    "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `false`.\n"
+                    "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `false`.\n",
```
Comment (guineveresaenger) [inline provider/cmd/pulumi-resource-aws/schema.json:279470]:
```suggestion
                    "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `true`.\n",
```

File: provider/cmd/pulumi-resource-aws/schema.json [hunk 2]
```diff
@@ -279522,7 +279524,8 @@
...
```
Comment (guineveresaenger) [inline provider/cmd/pulumi-resource-aws/schema.json:279527]:
```suggestion
                    "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `true`.\n",
```

File: provider/cmd/pulumi-resource-aws/schema.json [hunk 3]
```diff
@@ -279562,7 +279565,8 @@
                     },
                     "forceDelete": {
                         "type": "boolean",
-                        "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `false`.\n"
+                        "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `false`.\n",
```
Comment (guineveresaenger) [inline provider/cmd/pulumi-resource-aws/schema.json:279568]:
```suggestion
                        "description": "If `true`, will delete the repository even if it contains images.\nDefaults to `true`.\n",
```

**Review by guineveresaenger — CHANGES_REQUESTED** [review]
unless I'm missing some context, I think you want to change the comment to say it defaults to `true`, right?

**Comment by stack72** [comment]
ok @guineveresaenger based on your comments here, we need to shelve this - this comes from the TF provider not from us

**Comment by guineveresaenger** [comment]
welp, I didn't even see my comments were on `schema.json`. It's been a long week...

**Pull request closed without merging**